packages/pytest-simcore/src/pytest_simcore/helpers/typing_env.py

```diff
@@ -1,7 +1,8 @@
+from collections.abc import Iterable
 from typing import TypeAlias
 
 EnvVarsDict: TypeAlias = dict[str, str]
-EnvVarsList: TypeAlias = list[str]
+EnvVarsList: TypeAlias = Iterable[str]
```

Comment:
not sure I get why this becomes an iteratble instead of a list? is this to work with sets or so?

Reply:
It means "anything that can be iterated". More precisely https://docs.python.org/3/library/collections.abc.html#collections-abstract-base-classes

Yes, the naming is not right though